docs/source/tertiary/regression-tests.rst

```diff
@@ -110,211 +110,156 @@ For complete parameter usage information, check out the API reference for :func:
+
+  # genotype_states returns the number of alt alleles for each sample
+  # mean_substitute replaces any missing genotype states with the mean of the non-missing states
+  genotypes = (glow.transform('split_multiallelics', variants)
+    .withColumn('gt', glow.mean_substitute(glow.genotype_states(col('genotypes'))))
+    .select('contigName', 'start', 'names', 'gt')
+    .cache())
+
+  # Read covariates from a CSV file
+  covariates = pd.read_csv(covariates_csv, index_col=0)
+
+  # Read phenotypes from a CSV file
+  phenotypes = pd.read_csv(binary_phenotypes_csv, index_col=0)
+
+  # Run logistic regression test with approximate Firth correction for p-values below 0.05
+  log_reg_df = glow.gwas.logistic_regression(
+    genotypes,
+    phenotypes,
+    covariates,
+    correction='approx-firth',
+    pvalue_threshold=0.05,
+    values_column='gt'
   )
 
-  # Firth test
-  firth_log_reg_df = genotypes.select(
-    'contigName',
-    'start',
-    'names',
-    glow.expand_struct(glow.logistic_regression_gwas(
-      col('gt'),
-      lit(phenotype),
-      lit(covariates.to_numpy()),
-      'Firth'
-    ))
+.. invisible-code-block: python
+
+  expected_corrected_row = Row(
+    contigName='21',
+    start=9411301,
+    names=['rs531010746'],
+    effect=1.6771295643163748,
+    stderror=0.727886866796042,
+    correction_succeeded=True,
+    tvalue=7.561977300936178,
+    pvalue=0.005961255160993639,
+    phenotype='Binary_Trait_2'
   )
 
-  # Logistic regression with offset
-
-  # Read offset from a csv file
-  offset = np.hstack(pd.read_csv(binary_offset_csv, index_col=0)[[trait]].to_numpy()).astype('double')
-
-  # LRT test with offset
-  lrt_log_reg_with_offset_df = genotypes.select(
-    'contigName',
-    'start',
-    'names',
-    glow.expand_struct(glow.logistic_regression_gwas(
-      col('gt'),
-      lit(phenotype),
-      lit(covariates.to_numpy()),
-      'LRT',
-      lit(offset)
-    ))
+  expected_uncorrected_row = Row(
+      contigName='22',
+      start=16050114,
+      names=['rs587755077'],
+      effect=None,
+      stderror=None,
+      correction_succeeded=None,
+      tvalue=2.8620926885005393,
+      pvalue=0.09068944361066741,
+      phenotype='Binary_Trait_1'
+    )
+
+  assert_rows_equal(
+    log_reg_df.filter("contigName = 21 and start = 9411301 and phenotype = 'Binary_Trait_2'").head(),
+    expected_corrected_row
   )
 
-  # Firth test with offset
-  firth_log_reg_with_offset_df = genotypes.select(
-    'contigName',
-    'start',
-    'names',
-    glow.expand_struct(glow.logistic_regression_gwas(
-      col('gt'),
-      lit(phenotype),
-      lit(covariates.to_numpy()),
-      'Firth',
-      lit(offset)
-    ))
+  assert_rows_equal(
+    log_reg_df.filter("contigName = 22 and start = 16050114 and phenotype = 'Binary_Trait_1'").head(),
+    expected_uncorrected_row
   )
 
+The logistic regression function accepts GloWGR phenotypic predictions (either global or per chromosome) as an offset.
```

Comment:
This sentence reads along with your tip below is a bit repetitive. I suggest we give this offset thing their own subsection in this page both under linear and under logistic regression. I think it is important enough to get its own subsections.